composer.json

```diff
@@ -44,6 +44,7 @@
         "symfony/yaml": "~2.3",
         "symfony/security-acl": "~2.4",
         "sonata-project/intl-bundle": "~2.1",
+        "sllh/php-cs-fixer-styleci-bridge": "~1.0",
```

Comment:
This bridge unfortunately doesn't work, because styleci's symfony preset is not the same as php-cs-fixer's symfony level, and we also have extra fixers not available in php-cs-fixer.